setlexsem/generate/generate_prompts.py

```diff
@@ -96,6 +264,8 @@ def main(config_file):
     config = read_config(config_file)
 
     # Experiment config
+    number_of_data_points = config["N_RUN"]
```

Comment:
This variable can use some more specificity, but leave it as is for now. I believe the ideas is that `number_of_data_points` is the number of times to render a given prompt with different samples of sets?

Reply:
No, it's the number of sets generated per its configuration. maybe it needs renaming too?